src/mlpack/methods/hmm/hmm_impl.hpp

```diff
@@ -632,8 +658,8 @@ void HMM<Distribution>::Backward(const arma::mat& dataSeq,
       for (size_t state = 0; state < transition.n_rows; state++)
       {
         backwardLogProb(j, t) = math::LogAdd(backwardLogProb(j, t),
-            logTrans(state, j) + backwardLogProb(state, t + 1)
-            + emission[state].LogProbability(dataSeq.unsafe_col(t + 1)));
+          logTrans(state, j) + backwardLogProb(state, t + 1) +
+            logProbs(state, t + 1));
```

Comment:
Style issue---this wrapped line should be doubly-indented.  (Looks like it wasn't before your changes but we can fix it now. :))